### ISSUE TITLE
test-wpt-failure suite is busted

### ISSUE THREAD

**Issue by jdm** [servo/servo#7055]
We removed wpt.sh, and that caused the execution to fail in a fail that we interpret as success, I guess:

```
godot2:servo jdm$ ./mach test-wpt-failure
bash: tests/wpt/run.sh: No such file or directory
godot2:servo jdm$ echo $?
0
```

cc @jgraham @Ms2ger

**Comment by metajack** [comment]
The script should probably use `set -e` to prevent the failure from being hidden.

**Comment by Ms2ger** [comment]
test-wpt-failure is supposed to test that we get a non-zero return value if a test fails. I asked jgraham to make it work again when he broke it.

**Comment by larsbergstrom** [comment]
Is this a potential `E-Easy` bug for infra?

**Comment by Ms2ger** [comment]
jdm fixed in #15181.
